src/js/videojs.record.js

```diff
@@ -1194,8 +1194,13 @@ class Record extends Plugin {
                 try {
                     var track = this.stream.getVideoTracks()[0];
                     var imageCapture = new ImageCapture(track);
+                    let photoSettings = {
+                        imageWidth: recordCanvas.width,
+                        imageHeight: recordCanvas.height
+                    };
 
-                    imageCapture.takePhoto().then((blob) => {
+                    // take picture
+                    imageCapture.takePhoto(photoSettings).then((blob) => {
```

Comment:
It's not work for me in chrome Version 68.0.3440.106 (Official Build) (64-bit), is it work for you?

Reply:
yes it does work here, same version of Chrome. Are you getting an error? please open a new ticket instead of commenting on pull requests. 
![image-only](https://user-images.githubusercontent.com/305679/44469109-07d2f000-a627-11e8-819b-a39e5c59ade2.png)